tests/basic/test_location_buttons.py

```
@@ -0,0 +1,330 @@
+        # Check list view button styling
+        list_view_button = page.locator("#listViewButton")
+        expect(list_view_button).to_have_css("opacity", "0.6")
+        expect(list_view_button).to_have_css("filter", "grayscale(1)")
+
+
+@REQUIRES_NEW_FRONTEND
+class TestLocationButtonsMobile:
+    """Test suite for location buttons on mobile devices"""
+
+    @pytest.mark.parametrize("device_name", ALL_MOBILE_DEVICES)
+    def test_list_view_shows_tooltip_on_tap(self, mobile_page: Page, device_name: str):
```

Ruff (0.14.11)

Unused method argument: `device_name`

(ARG002)